vlasiator.cpp

```diff
@@ -942,35 +944,44 @@ int simulate(int argn,char* args[]) {
                doNow[0] = 2; // Setting to 2 so as to not increment the restart count below.
                globalflags::writeRestart = false; // This flag is only used by MASTER_RANK here and it needs to be reset after a restart write has been issued.
             }
-         }
-         else {
+         } else {
             doNow[0] = 0;
```

Comment:
I would suggest, for clarity:

on line 935, just for master rank, set all elements of `doNow` to zero. 
Remove the `else`-things from 948 and 959.